pihole-google.txt

```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+android-safebrowsing.googleapis.com
+developers.google.com
+malware-error.mozilla.con
+phish-error.mozilla.con
```

Comment:
mozilla.con domains do not exist and are not linked to Google. These should be removed

Reply:
All changes made